tests/phpunit/tests/media.php

```diff
@@ -2919,7 +2923,7 @@ public function test_wp_filter_content_tags_loading_lazy_opted_in() {
 			%2$s';
 
 		$content_unfiltered = sprintf( $content, $img, $iframe );
-		$content_filtered   = sprintf( $content, $lazy_img, $lazy_iframe );
+		$content_filtered   = sprintf( $content, wp_img_tag_add_decoding_async_attr( $lazy_img ), $lazy_iframe, 'the_content' );
```

Comment:
My prior suggestion was in error. This should fix the tests now:

```suggestion
		$content_filtered   = sprintf( $content, wp_img_tag_add_decoding_async_attr( $lazy_img, 'the_content' ), $lazy_iframe );
```